tasks/main.yml

```diff
@@ -79,10 +79,22 @@
     mode: 0770
   become: "{{ deploydrupal_files_become }}"
 
+# See drush cache clear below for additional details.
+- name: clear caches.
+  shell: "{{ deploydrupal_drush_path }} cache-rebuild"
+  args:
+    chdir: "{{ deploydrupal_core_path }}/sites/{{ deploydrupal_site_name }}"
+  when:
+    - deploydrupal_drush_cache_rebuild
+
+# Drush ^9 uses the Drupal service container. Clearing Drush cache before Drupal
+# cache can lead to fatal errors due to stale cache in the service container.
 - name: clear drush cache.
   shell: "{{ deploydrupal_drush_path }} cache-clear drush"
   args:
     chdir: "{{ deploydrupal_core_path }}/sites/{{ deploydrupal_site_name }}"
+  when:
```

Comment:
I added this as well. Seemed to make sense. Thoughts?

Reply:
I don't think we want any conditional on this, right? The previous drush cache clear did not have one.

Reply:
This is on our other cache rebuild.

https://github.com/ChromaticHQ/ansible-deploy-drupal/blob/master/tasks/main.yml#L158-L163

I'm a bit at a loss here too, but if we don't want it to run there, it shouldn't run here either. I assume this is to allow for a deployment without a cache clear, so we wouldn't want it anywhere, right?

Reply:
Yes. I didn't realize it was a pre-existing conditional. This looks good then.